src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -107,8 +108,20 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        $container = new ContainerBuilder();
+        $willBeAvailable = static function (string $package, string $class, string $parentPackage = null) use ($container) {
+            $parentPackages = (array) $parentPackage;
```

Comment:
`static $container; $container ??= new ContainerBuilder();` wdyt?

Reply:
`Parse error: syntax error, unexpected '='` on PHP 7.2...

Reply:
maybe the container a static variable would also be a totally different behavior